lib/chewy/rake_helper.rb

```diff
@@ -47,7 +47,7 @@ def normalize_indexes *indexes
       def reset_index *indexes
         normalize_indexes(indexes).each do |index|
           puts "Resetting #{index}"
-          index.reset! (Time.now.to_f * 1000).round
+          index.reset! (Time.now.to_f * 1000).round, journal: false
```

Comment:
We can actually do journal: false a default option for the reset! method, but is should be redefinable for sure.